src/token-list.json

```diff
@@ -1706,6 +1706,19 @@
       "coingeckoId": "sator",
       "website": "https://sator.io"
     }
+  },  
+  {
+    "chainId": 101,
+    "address": "YAWtS7vWCSRPckx1agB6sKidVXiXiDUfehXdEUSRGKE",
+    "symbol": "YAW",
+    "name": "Yawww",
+    "decimals": 6,
+    "logoURI": "https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/YAWtS7vWCSRPckx1agB6sKidVXiXiDUfehXdEUSRGKE/yaw.png",
+    "tags": [
+      "utility-token"
+    ],
+    "extensions": {
+      "coingeckoId": "yawww"
   },
```

Comment:
@OplopUser Looks like you're missing a curly brace here `}`

Reply:
thanks fixed it in the branch